src/main/java/com/google/growpod/servlets/UserServlet.java

```diff
@@ -114,9 +115,12 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
         response.getWriter().println(new Gson().toJson(list));
         return;
       }
+
+      response.getWriter().println("{ \"status\": \"ok\", \"value\": \"test\" }");
+
+      response.sendError(
+          HttpServletResponse.SC_METHOD_NOT_ALLOWED, "Unimplemented: " + request.getRequestURI());
```

Comment:
Will this send both "status: ok" and the 4xx not allowed in the same response?

Reply:
It seems so, Cody wrote this section of the code so I'm assuming that maybe this is there for testing or debugging purposes.I'll ask him if this still needs to be included.

Reply:
I'd say line 119 -- this is not useful here at least at first glance. It's absent in my copy of the file.